components/inputForm/inputForm.js

```diff
@@ -0,0 +1,52 @@
+import styled from "styled-components";
+
+export default function InputForm() {
```

Comment:
You're missing the prop `onClick` which you need to pass down to the form. Currently, your form refreshes the window and does nothing.